olive/passes/onnx/transformer_optimization.py

```diff
@@ -77,9 +76,8 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
                 description="Optimization options that turn on/off some fusions.",
             ),
             "opt_level": PassConfigParam(
-                type_=Any,
-                default_value=None,
-                searchable_values=Categorical([0, 1, 2, 99]),
+                type_=int,
+                default_value=2,
```

Comment:
should we keep the default value as `None`? When it is None, the optimizer uses the default value recommended for the model type 
https://github.com/microsoft/onnxruntime/blob/291a5352b27ded5714e5748b381f2efb88f28fb9/onnxruntime/python/tools/transformers/optimizer.py#L292
https://github.com/microsoft/onnxruntime/blob/291a5352b27ded5714e5748b381f2efb88f28fb9/onnxruntime/python/tools/transformers/optimizer.py#L50